packages/scene-composer/src/components/SceneComposerInternalSnap.spec.tsx

```diff
@@ -0,0 +1,197 @@
+import * as React from 'react';
+import renderer, { act } from 'react-test-renderer';
+import str2ab from 'string-to-arraybuffer';
+import flushPromises from 'flush-promises';
+
+import { SceneComposerInternal, COMPOSER_FEATURES, setFeatureConfig } from '..';
+import * as SceneLayoutComponents from '../layouts/SceneLayout';
+import { invalidTestScenes, testScenes } from '../../tests/testData';
+
+jest.mock('../layouts/StaticLayout', () => ({
+  StaticLayout: 'StaticLayout',
+}));
+
+jest.mock((window as any).ResizeObserver, () => ({
```

Comment:
I have no idea off hand.  I renamed this file to make it clearer what was snapshot and what was API tests files since the APIs needed different mocking to not break the snapshots.